test/functional/sqlsrv/0052.phpt

```diff
@@ -39,8 +40,8 @@ scrollable results with no rows.
     if ($row === false) {
         print_r(sqlsrv_errors(), true);
     }
-
-    $stmt = sqlsrv_query($conn, "SELECT * FROM ScrollTest");
+    
+    $stmt = AE\selectFromTable($conn, $tableName);
```

Comment:
Do we need to use a common function for a one liner? I think it's a lot more clear to use `$stmt = sqlsrv_query($conn, "SELECT * FROM ScrollTest");` instead of `$stmt = AE\selectFromTable($conn, $tableName);`. Same with line 29.

Reply:
When AE is enabled we have to select using sqlsrv_prepare/execute()

Reply:
why is that? you're selecting and you're not selecting a max column

Reply:
I know, but I want all refactored tests to be consistently using AE helper methods to replace the existing sqlsrv_query()

Reply:
I'm not too sure about this. That means you'll always be using sqlsrv_prepare and sqlsrv_execute for all queries when isColEncrypted is true even when prepare/execute isn't really required, and sqlsrv_query is never tested (or rarely tested) when column encryption is enabled.

Reply:
Based on issue #569 we are supposed to handle the combination of AE and sqlsrv_query(). Do you want sqlsrv_query() to check if max column is in the query? Or do we simply discourage using sqlsrv_query() when AE is enabled?

Reply:
Plus, some existing KSP tests do use sqlsrv_query() to fetch data. I think we need to reconsider how to tackle issue 569.